src/lib/autoLauncher.ts

```diff
@@ -43,50 +43,40 @@ export class LinuxAutoLauncher {
       Exec=${this.appPath}${hiddenArg}
       Icon=${this.appName}
     `;
-    await this.fs.writeFile({ contents, path: autostartAppFile }).catch((error) => {
-      errorLogger(error);
-    });
+    await this.fs.writeFile({ contents, path: autostartAppFile });
     response.stdout.push('success');
     return response;
   }
 
   public async disable(): Promise<ChildReturnData> {
-    const autostartAppFile = await this.getAutostartFilePath();
+    const autostartAppFile = this.getAutostartFilePath();
     const response: ChildReturnData = { stderr: [], stdout: [] };
-    await this.fs.removeFile(autostartAppFile).catch((error) => {
-      errorLogger(error);
```

Comment:
why are we removing `catch(error)`? Are we sure it won't produce any errors? I see in many of the below changes, `catch(error)` is deleted along with the `try` block. Couldn't understand what's happening tbh

Reply:
in order to propagate error, otherwise, `catch` will swallow it